rpm_spec/subpackages/manageiq-core

```diff
@@ -32,11 +35,16 @@ done
 
 %post core
 %{__cp} -f %{app_root}/config/cable.yml.sample %{app_root}/config/cable.yml
+# for upgrades, fix up previously created certs
+%{__chmod} g+r %{app_root}/certs/v2_key
```

Comment:
Should this use `%attr`?

Reply:
`v2_key` is not managed by rpm. So the file is either created before or after this rpm is installed.

### before

We are upgrading this rpm and need to make sure the file is setup correctly.

- `%post` is run on the target system (vs the build system). we need to run `chown` and `chmod` to fix the file.

### after

Probably created at first boot `/bin/appliance-initialize.sh`, but could have been created via `scp`.

- `%pre` (and to some extend `%files` / `%attr`) set `SetGID` bit on `certs/` to make sure newly created files in this directory gets the group `manageiq`.
- currently no guarantee around `g+r` bit - I will need to do this in the kickstart for the default case